bin/create

```diff
@@ -4,7 +4,8 @@ set -eu
 
 ROOT=$SNAP_COMMON/classic
 
-if [ -d $ROOT ]; then
+# check for the last bit that got created
+if [ -x $ROOT/usr/sbin/policy-rc.d ]; then
```

Comment:
Perhaps we ought to look for some special stamp file instead. While it is unlikely policy-rc.d goes away it seems like a more reliable approach.